src/Backups/RestorerFromBackup.h

```diff
@@ -77,6 +80,7 @@ class RestorerFromBackup : private boost::noncopyable
     BackupPtr backup;
     ContextMutablePtr context;
     QueryStatusPtr process_list_element;
+    std::function<void()> after_each_task_callback;
```

Comment:
maybe we can call it `after_task_callback`, what do you think ?

Reply:
I've renamed it.